app/templates/Users/index.php

```diff
@@ -23,12 +23,12 @@
                 <tr>
                     <td><?= $this->Number->format($user->id) ?></td>
                     <td><?= h($user->email) ?></td>
-                    <td><?= h($user->created) ?></td>
-                    <td><?= h($user->modified) ?></td>
+                    <td><?= h($user->created->format(DateTimeInterface::ATOM)) ?></td>
+                    <td><?= h($user->modified->format(DateTimeInterface::ATOM)) ?></td>
                     <td class="actions">
-                        <?= $this->Html->link(__('View'), ['action' => 'view', $user->id]) ?>
-                        <?= $this->Html->link(__('Edit'), ['action' => 'edit', $user->id]) ?>
-                        <?= $this->Form->postLink(__('Delete'), ['action' => 'delete', $user->id], ['confirm' => __('Are you sure you want to delete # {0}?', $user->id)]) ?>
+                        <?= $this->Html->link(__('View'), ['action' => 'view', $user->slug]) ?>
+                        <?= $this->Html->link(__('Edit'), ['action' => 'edit', $user->slug]) ?>
+                        <?= $this->Form->postLink(__('Delete'), ['action' => 'delete', $user->slug], ['confirm' => __('Are you sure you want to delete # {0}?', $user->slug)]) ?>
```

Comment:
I think you've misunderstood the comment https://github.com/sergio9977/tutorial-cakephp/pull/3#discussion_r1386584249

From the comment above, I highlighted the problem with article->id in `templates/Users/view.php`. To be exact at LOC below:

https://github.com/sergio9977/tutorial-cakephp/blob/2c14c39820f7ff5d6d69ad4be2b890361aa8a414/app/templates/Users/view.php#L65-L67

Current changes will break the user's screen. The field slug does not exist in the entity User. It only exists in entity Article. 

Please be careful when making changes to the part of the code and please ensure the changed part is working as intended.

Reply:
Sorry I forgot this part, I already fixed it.